examples/rllib/self_play_train.py

```diff
@@ -16,41 +16,35 @@
 import copy
 import os
 
-from ml_collections import config_dict
-import multiagent_wrapper  # pylint: disable=g-bad-import-order
 import ray
 from ray.rllib.agents.registry import get_trainer_class
+from ray.rllib.policy.policy import PolicySpec
+from ray.tune import tune
 from ray.tune.registry import register_env
 
+from examples.rllib.utils import env_creator
 from meltingpot.python import substrate
 
 
 def main():
-  # We need the following 3 pieces to run the training:
+  # We need the following 2 pieces to run the training:
   # 1. The agent algorithm to use.
   agent_algorithm = "PPO"
   # 2. The name of the MeltingPot substrate, coming
   # from substrate.AVAILABLE_SUBSTRATES.
-  substrate_name = "allelopathic_harvest"
-  # 3. The number of CPUs to be used to run the training.
-  num_cpus = 1
-
-  # function that outputs the environment you wish to register.
-  def env_creator(env_config):
-    env = substrate.build(config_dict.ConfigDict(env_config))
-    env = multiagent_wrapper.MeltingPotEnv(env)
-    return env
+  substrate_name = "clean_up"
```

Comment:
Why change the example substrate? If this is just because of initialisation time (Allelopathic harvest is not the fastest), we can choose Bach or Stravinsky in the Matrix, as that one is fast, and doesn't get zero rewards by default.